src/Serilog.Sinks.RollingFile/project.json

```diff
@@ -9,16 +9,17 @@
     "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png"
   },
   "dependencies": {
-    "Serilog": "2.0.0",
-    "Serilog.Sinks.File": "2.0.0"
+    "Serilog.Sinks.File": "3.0.0-dev-00735"
```

Comment:
`dotnet pack` will (should) fail on this, so if we forget it, it'll be caught by CI on the dev-master PR :+1: